rustworkx-core/Cargo.toml

```
@@ -19,15 +19,12 @@ rand_pcg = "0.3.1"
 rayon = "1.6"
 num-traits = "0.2"
 priority-queue = "1.2"
-rayon-cond = "0.2"
+rayon-cond = "0.3"
```

```suggestion
rayon-cond = "0.2"
```

I think this is the culprit